pkgdown/_pkgdown.yml

```diff
@@ -82,3 +82,10 @@ reference:
 - title: Data
   contents:
   - rhinos
+
+news:
+  releases:
+  - text: "Version 1.1.1"
+    href: https://github.com/Appsilon/rhino/releases/tag/v1.1.1
+  - text: "Version 1.1.0"
+    href: https://github.com/Appsilon/rhino/releases/tag/v1.1.0
```

Comment:
I like your idea of having release blog posts and linking them here! Until then let's remove this section - this will get us a simple "Changelog" link at the top (without a menu). I think this will give our users a cleaner experience - the links to GitHub releases are not very useful. Instead we can add them to the `NEWS.md` (see my other comment).
```suggestion
```

Reply:
These should be removed now.